### DISCUSSION TITLE
please enable transactions. connecting wallets alone is not enough

### DISCUSSION THREAD

**Discussion by Baliruno** [rainbow-me/rainbowkit#1425] [Feedback]
please enable transactions. connecting wallets alone is not enough

**Comment by DanielSinclair** [comment]
@Baliruno Wagmi is your friend for this! RainbowKit was built on top of Wagmi, and merely simplifies the connecting process, while Wagmi handles managing the account and signing/sending transactions. You can reference our [Next.js NFT Minting dApp](https://github.com/rainbow-me/rainbowkit/tree/main/examples/with-next-mint-nft) example to see what a full-stack dApp looks like. Also reference Wagmi's docs and hooks like [useSignMessage](https://wagmi.sh/react/hooks/useSignMessage) to see how you can request signatures from your users. As long as you've installed RainbowKit and Wagmi properly, you can use these hooks anywhere in your dApp.
